### ISSUE TITLE
data pre-processing before finetuning, divide by 255?

### ISSUE THREAD

**Issue by qiyang77** [kratzert/finetune_alexnet_with_tensorflow#79]
Hello, I use python 3.6 + tensorflow 1.9.0 to do a classification task by finetuning Alexnet. In kratzert's srouce code, the range of data is [0,255] and the image is substracted 
 by meanfile[103,116,123].  However, in my case, I perform better accrucy when the input images divide by 255 after substracted mean(distribution is [-0.5,0.5]). If the input images did not divide by 255 (distrubution is [-128, 128]), the results is bad and more easily to suffer Nan problem in summary operation when the learning rate is higher then 0.001.
The question is should I scale the image ?

**Comment by kratzert** [comment]
You probably should, yes. Not sure why I missed to do this here, since I'm doing this usually. But better than dividing by 255 you should divide by the per-channel standard deviation.

**Comment by qiyang77** [comment]
Thank u very much!
